yarn.lock

```diff
@@ -10940,7 +10940,6 @@ html-void-elements@^1.0.0:
 html-webpack-plugin@^5.4.0:
   version "5.5.0"
   resolved "https://registry.yarnpkg.com/html-webpack-plugin/-/html-webpack-plugin-5.5.0.tgz#c3911936f57681c1f9f4d8b68c158cd9dfe52f50"
-  integrity sha512-sy88PC2cRTVxvETRgUHFrL4No3UxvcH8G1NepGhqaTT+GXN2kTamqasot0inS5hXeg1cMbFDt27zzo9p35lZVw==
```

Comment:
are you sure you want to remove this?

Reply:
I had no idea why😅 Thanks for pointing out